web/src/message_list_view.ts

```diff
@@ -1655,7 +1658,9 @@ export class MessageListView {
             preserved_properties,
         );
 
-        const $rendered_recipient_row = $(render_recipient_row(group));
+        const current_filter = narrow_state.filter();
+        const channel_view = current_filter?.can_show_next_unread_topic_conversation_button();
```

Comment:
This might happen to do a valid calculation, but it is super confusing. 

Why aren't you calling `is_channel_view`?